src/usage.ts

```diff
@@ -120,5 +120,11 @@ export async function renderUsage<T extends ArgsDef = ArgsDef>(
     );
   }
 
+  if (version) {
+    usageLines.push(
+      `Use \`${commandName} --version\` to get the version of the CLI.`,
```

Comment:
I am thinking it could be displayed just after list of other arguments with same formatting.

Reply:
Something like this? 

![image](https://github.com/unjs/citty/assets/45267552/f1e03e1b-1dab-4f7c-a2a8-f252793a75e0)

Reply:
If yes, I'll push the code!

Reply:
Yes please something like this :)

Reply:
Ok, code is pushed!